code/modules/client/preferences/story_participant.dm

```diff
@@ -0,0 +1,5 @@
+/datum/preference/toggle/story_pref
+	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
+	default_value = TRUE
```

Comment:
Off by default. Most people are going to ignore this because they don't like being directed how to roleplay.

Reply:
i'm not turning it off by default, that will massively reduce uptake of the feature